meDsuMonitor

```diff
@@ -0,0 +1,12 @@
+os.loadAPI("M3C/utils")
+os.loadAPI("M3C/utilDsu")
+
+while true do
+  term.clear()
+  term.setCursorPos(1,1)
+
+  utilDsu.dsu.setDsus(utils.findAndWrapAll("deep_storage_unit"))
```

Comment:
This syntax seems to be a bit redundant. If we're in utilDsu, then I would assume we're already working with DSU's. So really it would probably be more sensible to structure it in a fashion where `utilDsu.setDsus()` works. Saying `utilDsu.dsu....` should be unnecessary.